ansys/mapdl/core/mapdl_grpc.py

```diff
@@ -18,27 +18,34 @@
 from tqdm import tqdm
 from grpc._channel import _InactiveRpcError, _MultiThreadedRendezvous
 
+
+msg_import = """There was a problem importing the ANSYS API module (ansys.api.mapdl).
```

Comment:
This should be all caps, denoting it's a [constant](https://www.python.org/dev/peps/pep-0008/#constants).